Formula/r/reuse.rb

```diff
@@ -0,0 +1,94 @@
+class Reuse < Formula
+  include Language::Python::Virtualenv
+
+  desc "Tool for copyright and license recommendations"
+  homepage "https://reuse.software"
+  url "https://github.com/fsfe/reuse-tool/archive/refs/tags/v5.0.2.tar.gz"
+  sha256 "a28c23bce1581a338db056cfc3e196cbe5b1701f5e2944a2c61afa50fb1d4954"
+  # License summary from project README (April 2024):
+  # - Original source code: GPL-3.0-or-later.
+  # - Documentation: CC-BY-SA-4.0.
+  # - Some config and data files: CC0-1.0.
+  # - Some code from spdx/tools-python: Apache-2.0.
+  # Verified: `reuse spdx | awk '/LicenseInfoInFile/{print $2}' | sort --unique`
```

Comment:
I think we can remove this, since the info is in the PR(s)

Reply:
Done!